vault/identity_store_util.go

```diff
@@ -1183,6 +1190,7 @@ func (i *IdentityStore) sanitizeAndUpsertGroup(ctx context.Context, group *ident
 		}
 	}
 
+alias:
```

Comment:
I have a slight preference for ALLUPPERCASE to make it really obvious it's a label. But either way is fine!

Reply:
Fixed.